website/blog/2023-02-07-automate-schema-evolution-at-scale-with-apache-hudi-in-aws-glue.mdx

```diff
@@ -1,7 +1,9 @@
 ---
 title: "Automate schema evolution at scale with Apache Hudi in AWS Glue | Amazon Web Services"
-authors:
-- name: Subhro Bose, Eva Fang and Ketan Karalkar
+authors: 
+- name: Subhro Bose
+- name: Eva Fang
+- name: Ketan Karalkar
 category: blog
 image: /assets/images/blog/automate-schema-evolution-at-scale-with-apache-hudi-in-aws-glue.png
```

Comment:
The current image associated with this blog is at assets/images/automate-schema-evolution-at-scale-with-apache-hudi-in-aws-glue.png . Fix the image location to fix the thumbnail issue.

Reply:
fixed-thanks